distributions/kfdef/kfctl_openshift_v1.3.0_servicemesh.yaml

```diff
@@ -0,0 +1,96 @@
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/openshift/openshift-scc/base
+      name: openshift-scc
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/cert-manager
+      name: cert-manager
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/admission-webhook/overlays/servicemesh
+      name: admission-webhook
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/openshift-service-mesh
+      name: istio-managed
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/knative/overlays/servicemesh
+      name: knative
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/jupyter-web-app/overlays/servicemesh
+      name: jupyter-web-app
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/notebook-controller/overlays/servicemesh
+      name: notebook-controller
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/centraldashboard/overlays/servicemesh
+      name: centraldashboard
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/volumes-web-app/overlays/servicemesh
+      name: volumes-web-app
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/pytorch-job/overlays/servicemesh
+      name: pytorch-job
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/tf-job/overlays/servicemesh
+      name: tf-job
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/katib/overlays/servicemesh
+      name: katib
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/kfp-tekton/overlays/servicemesh
+      name: kfp-tekton
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/kfserving/overlays/servicemesh
+      name: kfserving
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift
+      name: kubeflow-apps
+  repos:
+    - name: odhmanifests
+      uri: 'https://github.com/VaishnaviHire/odh-manifests/archive/integrate_service_mesh.tar.gz'
+    - name: manifests
+      uri: 'https://github.com/VaishnaviHire/manifests/archive/test_sm_istio.tar.gz'
+  version: v1.3.0-tag
```

Comment:
The uri need to change to point to a release with a branch name or tag, not your repo.